src/core.js

```diff
@@ -40,7 +40,14 @@ export default class Base extends EventEmitter {
       'authorization_error',
       'hash_parsed',
       'signin ready',
-      'signup ready'
+      'signup ready',
+
+      'forgot_password ready',
+      'signin submit',
+      'signup submit',
+      'forgot_password submit',
+      'last_login ready',
```

Comment:
this array is exposed to `lock.validEvents`. if we remove this, the analytics script won't work as expected.